src/applications/vaos/utils/selectors.js

```diff
@@ -431,9 +431,8 @@ export const selectPastAppointments = createSelector(
   },
 );
 
-export function selectHasExpressCareRequests(state) {
-  return (
-    vaosExpressCare(state) &&
-    state.appointments.future?.some(appt => appt.vaos.isExpressCare)
-  );
-}
+export const selectExpressCare = state => ({
+  ...state.appointments?.expressCare,
+  enabled: vaosExpressCare(state),
+  hasRequests: state.appointments.future?.some(appt => appt.vaos.isExpressCare),
```

Comment:
I think we still want the feature flag check in here, otherwise we'll hide the headers on the upcoming/past tabs when there are EC requests but we haven't turned on the new tab.